configure.in

```diff
@@ -8725,10 +8733,14 @@ for subdir in $SUBDIRS; do
                 done
             elif test ${subdir} = "utils"; then
                 makefiles=""
-                for util in ifacecheck ; do
+                for util in ifacecheck wxrc ; do
                     if test -d $srcdir/utils/$util ; then
-                        makefiles="utils/$util/src/Makefile.in \
-                                   $makefiles"
+                        if test -d $srcdir/utils/$util/src; then
```

Comment:
This is a nitpick, but it took me some time to understand what was going on here. I think using `test -f ...path.../Makefile.in` would make this more clear as it would read more naturally: if this makefile exists, then add it, otherwise add that one.